internal/applicationsnapshot/report.go

```diff
@@ -289,6 +309,67 @@ func generateMarkdownSummary(r *Report) ([]byte, error) {
 	return markdownBuffer.Bytes(), nil
 }
 
+func generateTextReport(r *Report) ([]byte, error) {
+
+	mainTemplate := hd.Doc(`
+		Results
+		=======
+
+		{{ range .Components -}}
+		Name:       {{ .Name }}
+		ImageRef:   {{ .ContainerImage }}
+		Summary:    Violations: {{ len .Violations }}, Warnings: {{ len .Warnings }}, Successes: {{ .SuccessCount }}
+		{{ if gt (len .Violations) 0 -}}
+		Violations: {{ template "results" .Violations }}
+		{{ end }}
+		{{ if gt (len .Warnings) 0 -}}
+		Warnings: {{ template "results" .Warnings }}
+		{{ end }}
+		{{ if and (gt .SuccessCount 0) (.Successes) -}}
+		Successes: {{ template "results" .Successes }}
+		{{ end }}
+		{{ end }}{{/* range .Components */}}
+	`)
+
+	partials := map[string]string{
+		// For convenience since juggling the {{- and -}} gets tricky
+		"NL": "\n",
```

Comment:
Yes, though much less so once you get into the rhythm of using only/mostly `{{-`, see the [templates for the docs](https://github.com/enterprise-contract/ec-cli/tree/main/internal/documentation/asciidoc) for inspiration

Reply:
Actually I didn't end up using it :sweat_smile: